src/styles/index.scss

```
@@ -1,3 +1,119 @@
+@import './variables';
+@import './stars';
+@import 'https://fonts.googleapis.com/css?family=Roboto:regular,500,700';
+
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  line-height: 14px;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  background-color: grey;
+  color: $greyColor;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 50px 120px;
+  width: 200px;
+}
+
+.card-block {
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__img img {
+    box-sizing: border-box;
```

Do not increase specificity, move `box-sizing: border-box;` into `*` selector
```suggestion
  &__img{
```